### PR TITLE
Adding google secret manager iam module README.md

### PR DESCRIPTION
**Pull request by Hanymin**

### CODE REVIEW

**Review by mariux — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,285 @@
...
+  The resource name of the folder the policy is attached to. Its format is `folders/{folder_id}`.
+
+- **`members`**: **_(Required `set(string)`)_**
+
+  Identities that will be granted the privilege in role for `'google_secret_manager_secret_iam_binding'`. Each entry can have one of the following values: `'allUsers'`, `'allAuthenticatedUsers'`, `'serviceAccount:{emailid}'`, `'group:{emailid}'`, `'domain:{domain}'`."
+  Default is `[]`.
+
+- **`role`**: **_(Required `string`)_**
+
+  The role that should be applied. Only one `'google_secret_manager_secret_iam_binding'` can be used per role. Note that custom roles must be of the format `'[projects|organizations]/{parent-name}/roles/{role-name}'`.
```
Comment (mariux) [inline README.md:110]: this does not make sense in the context of the module..please rephrase

File: README.md [hunk 2]
```diff
@@ -1 +1,285 @@
...
+
+#### Main Resource Configuration
+
+- **`secret_id`**: **_(Required `string`)_**
+
+  The resource name of the folder the policy is attached to. Its format is `folders/{folder_id}`.
+
+- **`members`**: **_(Required `set(string)`)_**
+
+  Identities that will be granted the privilege in role for `'google_secret_manager_secret_iam_binding'`. Each entry can have one of the following values: `'allUsers'`, `'allAuthenticatedUsers'`, `'serviceAccount:{emailid}'`, `'group:{emailid}'`, `'domain:{domain}'`."
```
Comment (mariux) [inline README.md:105]: this does not make sense in context of the module... referencing a resource here should be avoided.

File: README.md [hunk 3]
```diff
@@ -1 +1,285 @@
...
+
+- **`project`**: _(Optional `string`)_
+
+  The ID of the project in which the resource belongs. If it is not provided, the provider project is used."
+  Default is `null`.
+
+- **`authoritative`**: _(Optional `bool`)_
+
+  Secret manager secret iam choice whether `'authoritative == true'` use `google_secret_manager_secret_iam_binding`, `'authoritative == false'` use `google_secret_manager_secret_iam_member"`.
+  Default is `true`.
```
Comment (mariux) [inline README.md:120]: please add a real description not code.

File: README.md (hunk 1 above)
Comment (Hanymin) [reply README.md:110]: this is the official documentation wording of terraform.
should I change it ?

Comment (mariux) [reply README.md:110]: yes, but we are a module using it in a different way... we use role for iam_member and iam_binding... so the text needs to be adjusted to reflect what is implemented in themodule with the authoritative flag.

**Pull request merged**